src/main/java/dataMgmt/MetadataManager.java

```diff
@@ -156,6 +184,29 @@ public void fileMoved(long fileID, String newFilePath) {
         fileIDtoFilePath.put(fileID, newFilePath);
     }
 
+    public void fileDeleted(long fileID) {
+        String filePath = fileIDtoFilePath.get(fileID);
+        if (filePath != null){
+            fileMetadataMap.remove(filePath);
+        }
+        fileIDtoFilePath.remove(fileID);
```

Comment:
well ideally you'd have to check both, but yeah i was just saying they were grouped incorrectly